src/beam_stats_msg_graphite.erl

```diff
@@ -78,7 +78,7 @@ to_iolist(
     ValueBin = integer_to_binary(Value),
     TimestampInt = timestamp_to_integer(Timestamp),
     TimestampBin = integer_to_binary(TimestampInt),
-    [PathIOList, <<" ">>, ValueBin, <<" ">>, TimestampBin].
+    [PathIOList, <<" ">>, ValueBin, <<" ">>, TimestampBin, <<"\n">>].
```

Comment:
Good catch! It reveals my dirty secret of only really using StatsD in production and never sending to Graphite directly. I should also add a test case for Graphite/statser. Thanks!